### ISSUE TITLE
Event when PLC connection state changes ?

### ISSUE THREAD

**Issue by abrasat** [killnine/s7netplus#100]
Is there any event on client side available, which is triggered when the PLC gets disconnected ?
Does the connection get automatically re-established, or must the client call Open() again?

**Comment by mesta1** [comment]
No, to check if it is connected you have to periodically check IsConnected property.
No, we don't do magics to reconnect. You have to call Close and Open again.

**Comment by abrasat** [comment]
I was not thinking about magics, but of an eventual AutoReconnect configuration parameter
